packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html

```diff
@@ -367,10 +379,18 @@
                     </div>
                     <div class="oui-tile__definition">
                         <div class="oui-tile__definition">
-                            <strong class="oui-tile__term d-block" data-translate="overthebox_detail_links">
+                            <strong
+                                class="oui-tile__term d-block"
+                                data-translate="overthebox_detail_links"
+                            >
                             </strong>
-                            <oui-tile-button data-ng-href="{{:: $ctrl.guidesLink }}" external>
-                                <span data-translate="overTheBox_detail_link_guide"></span>
+                            <oui-tile-button
+                                data-ng-href="{{:: $ctrl.guidesLink }}"
```

Comment:
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-tile-webcomponent--with-buttons
`oui-tile-button` supports `href` attribute

Reply:
Sorry but I don't understand your feedback. I use `href` for `oui-tile-button`. Those lines doesn't change, there are just more far in the code...